src/interface/src/app/signup/signup.component.html

```diff
@@ -66,7 +78,6 @@ <h1 class="signup-title">Create your account</h1>
         </mat-form-field>
 
         <mat-error *ngIf="errors.length > 0">
-          <p>Please correct the following errors:</p>
           <ul>
```

Comment:
Removed this because some errors in this section aren't user errors. It's clear that the list of errors are errors, so it seemed redundant at best, unhelpful at worst. This could also be replaced by something like "The following errors were encountered".